locust/static/locust.js

```diff
@@ -235,7 +235,7 @@ function update_stats_charts(){
 
 // init charts
 var rpsChart = new LocustLineChart($(".charts-container"), "Total Requests per Second", ["RPS", "Failures/s"], "reqs/s", ['#00ca5a', '#ff6d6d']);
-var responseTimeChart = new LocustLineChart($(".charts-container"), "Response Times (ms)", ["Median Response Time", "95% percentile"], "ms");
+var responseTimeChart = new LocustLineChart($(".charts-container"), "Response Times (ms)", [(percentile1 * 100).toString() + "% percentile" , (percentile2 * 100).toString() + "% percentile"], "ms");
```

Comment:
Can you change this to `..th percentile` instead of `..% percentile`?

Reply:
Updated locust.js. also I have updated report.html

![image](https://user-images.githubusercontent.com/79035645/224551504-37fe210f-b73c-488c-8173-1126fab6f5f6.png)